demos/disco.sh

```diff
@@ -60,7 +60,7 @@ registry rpc get-api-spec \
 registry rpc get-api-spec --json \
 	--name projects/$PROJECT/locations/global/apis/translate/versions/v3/specs/discovery.json
 
-# You might notice that that didn't return the actual spec. That's because the spec contents
+# You might notice that didn't return the actual spec. That's because the spec contents
```

Comment:
doubled that is correct more often than most doubled words, but it often isn't strictly necessary, and this appears to be such a case. Happy to drop if I'm wrong.

Reply:
Yup, it's correct both ways, so less is more!